server.js

```diff
@@ -0,0 +1,27 @@
+const express = require("express");
+
+// Entry point or our application
+async function main() {
+	const app = express();
+
+	return app.listen(8080);
+}
+
+main().then(({ server, client }) => {
+  # Run the shutdown function when we receive a kill signal
```

Comment:
Ah, this was a typo in my markdown... this is meant to be a comment, but I was swapping between `sh` and `javascript` files